src/Testcontainers.WebDriver/Testcontainers.WebDriver.csproj

```diff
@@ -0,0 +1,14 @@
+<Project Sdk="Microsoft.NET.Sdk">
+    <PropertyGroup>
+        <TargetFrameworks>netstandard2.0;netstandard2.1</TargetFrameworks>
+        <LangVersion>latest</LangVersion>
+    </PropertyGroup>
+    <ItemGroup>
+        <PackageReference Include="JetBrains.Annotations" Version="2022.3.1" />
+        <PackageReference Include="Selenium.Support" Version="4.8.0" />
+        <PackageReference Include="Selenium.WebDriver" Version="4.8.0" />
```

Comment:
O.k
I need to think again if the responsible for mapping between the: RemoteWebDriver that charges on sending the request running test, and the WebDriverContainer is another user or another developer.
Because there is  [stereotype ](https://www.selenium.dev/documentation/grid/configuration/toml_options/#configuring-and-customising-drivers)
To fit between them.
And then it also covers the browsers issue

Reply:
Implementation in Java can help you to make your decision. Here is a [new getter method](https://github.com/testcontainers/testcontainers-java/blob/main/modules/selenium/src/main/java/org/testcontainers/containers/BrowserWebDriverContainer.java#L293) to get just Selenium URL. And [old method](https://github.com/testcontainers/testcontainers-java/blob/main/modules/selenium/src/main/java/org/testcontainers/containers/BrowserWebDriverContainer.java#L334) to get whole WebDriver is marked as obsolete. 
It has certain small disadvantage that video recording container starts immediately when Selenium container is in running state. It means that it records just Selenium logo till the  point when WebDriver session is created which can be in my case even several minutes. But I think with good API design it can be easily extended by developers.

Reply:
@vlaskal 
Sorry about the delay I was seak
For our purposes, I don't want to implement it like java, because they use the DriverOptions (chroneOptions and so on) 
Yes, I want to do the mapping between the Url of `new RemoteWebDriver(string URL,  DriverOptions options)` and `new WebDriverBuilder().Build()`
The next stage will be to connect it to the video recording cotnainer